_content/essays/jhanas-and-the-lay-disciple_bodhi.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Jhānas and the Lay Disciples: According to the Pāli Suttas"
+authors:
+  - bodhi
+external_url: "http://dhammatalks.net/Books10/Bhikkhu_Bodhi_The_Jhanas_and_the_Lay_Disciples.htm"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/1fKmtexkIEFaJJHZ64TUyka2VGMmthPKJ/view?usp=sharing"
+course: samadhi
+tags:
+  - theravada
+  - jhana
+  - stages
+pages: "14"
```

Comment:
BTW: The month and year here us Jun, 2004 😊

Reply:
So the month/year it was put up on the website is ok? I was thinking this article was originally published at a different time.